simplesat/constraints/tests/test_requirement.py

```diff
@@ -265,3 +271,173 @@ def test_repr(self):
 
         # Then
         self.assertMultiLineEqual(repr(requirement), r_repr)
+
+
+class TestRequirementTransformation(unittest.TestCase):
+
+    name = "MKL"
+    not_name = "shapely"
+    never_name = "basemap"
+
+    def run_transformations(self, constraints, transform, allow):
```

Comment:
the name of that function (and the `assert*`) made it a bit difficult for me to follow.

Normally, the convention for `assert*` methods is to have the signature as `assertFoo(result, expected_result, ...)`. I think it would help changing the signature to that convention.

Also, it is not obvious what `assertSingleMulti` would do without looking at the code, same for `assertTransformation`. Can we get more explicit names there as well ?